docs/source/reference/benchmark/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _benchmark-overview:
+
+Benchmark
+================================================
+
+SkyPilot supports **easy benchmarking of cloud resources**.
+With minimal effort, you can find the right cloud resource for your task and **save significant cost**.
```

Comment:
On reading this, my first thought is "what is benchmarking?" Maybe consider writing as:

```
SkyPilot allows **easy measurement of performance and cost of different kinds of cloud resources** through the benchmark feature.
With minimal effort, you can find the right cloud resource for your task that fits your performance goals and budget constraints.
```